os/network/config.pan

```diff
@@ -18,5 +19,11 @@ variable NETWORK_DEFAULT_GATEWAY ?= null;
 
 '/system/network/interfaces' ?= copy_network_params(NETWORK_PARAMS);
 
+
+# Disable management of resolv.conf by NetworkManager (EL8+)
+include if ( (OS_VERSION_PARAMS['family'] == 'el') && (OS_VERSION_PARAMS['majorversion'] >= "8") ) 'os/network/network_manager';
```

Comment:
@jouvin the nmstate PR from @aka7 has this managed by the nmstate module and using a different filename.
i'm ok that this is used for el8, but this should not come in each others ways

Reply:
@stdweird I am not sure it is a real problem. If we have 2 files defining the same setting with the same value it will not heart and it is better that this template doesn't overwrite what is done by the nmstate component. I think we need something urgently as I'm pretty sure that most EL8 systems will be hitted by the problem at the next REHEL version planned at the beginning of the autumn. If nmstate component is ready before, we could just use it. If there is a chance that it is not or not yet ready for prime time, I'd suggest to add this PR too.

Reply:
we can discuss this, maybe come up with a prefix for names used by components vs doing it in pan. I liked the idea of module managing it because switching becomes easier.  harder to cleanup with metaconfig when you want to remove it.

Reply:
I think I agree with both of you that the right place for fixing this is the network component and as soon as the `nmstate` variant of the network component is ready, I see no reason not to use it as the default component "backend" on EL8+ systems.

@aka7 I see your point on the difficulty to remove a file added by metaconfig but can easily make it an empty file which has the same effect. But as the issue I raised is probably still  a corner case (my feeling is that it doesn't affect the current RHEL8 version but I may be wrong as I am using CentOS Stream) and we can wait to see how fast `nmstate` variant is finalized. In the meantime, keeping this PR open may help other sites facing the problem to work around it...

Reply:
@jouvin @aka7 we can try to settle on the same filename and the same content. then this will be easier to handover to the nmstate module

Reply:
@stdweird as I said previously, I don't like this idea. The risk is that at some point the component (or the template) decide to add new things in the config file and that the other overwrites the changes. What we could do is agree on ordering so that the component wins. For example, the component could use `95-quattor.conf` and the component `98-quattor-nmstate.conf` so that the component one as a higher priority.

Reply:
@jouvin see, https://github.com/aka7/configuration-modules-core/blob/9253e89a2ab06b1d755219876c09670e766e6812/ncm-network/src/main/perl/nmstate.pm#L82
 I used 90-
happy to call something else.

Reply:
@jouvin i get that, but i think the idea is that is nmstate becomes viable, this workaround can go. 
@aka7 for nmstate  to have the final say, it needs to win over this file, and it should not delete the conf when the config sets networkmanager  to control the dns, it should make the config excplict (so don't delete the file, but set `dns=...`).

Reply:
@stdweird to set it back to default we have to just not set it then, so either can have dns= (empty line) or just [main] section with no option or value. looking at the man page it just seem to state that you can set dns=none to not manage dns, otherwise default is used.

so I've come up with this,
https://github.com/quattor/configuration-modules-core/pull/1601/commits/d244ea56dd941c181364acae9b0c20b1b10a6e9f